File: app/src/entourage/java/social/entourage/android/map/entourage/create/wizard/CreateActionWizardListener.java
package social.entourage.android.map.entourage.create.wizard;

/**
 * Created by Mihai Ionescu on 05/10/2018.
 */
public interface CreateActionWizardListener {

    void createActionWizardPreviousStep(int currentStep);
    void createActionWizardNextStep(int currentStep, int option);

}

File: app/src/entourage/java/social/entourage/android/entourage/create/wizard/CreateActionWizardListener.kt
package social.entourage.android.entourage.create.wizard

/**
 * Created by Mihai Ionescu on 05/10/2018.
 */
interface CreateActionWizardListener {
    fun createActionWizardPreviousStep(currentStep: Int)
    fun createActionWizardNextStep(currentStep: Int, option: Int)
}